config/istio/overlays/lock-it-down.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: pilot-network-policy
+  namespace: istio-system
+spec:
+  podSelector:
+    matchLabels:
+      app: pilot
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/istio-system-ns: ""
+    ports:
+    - port: 15004
+  - ports:
+    - port: 15010
+    - port: 15011
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: citadel-network-policy
+  namespace: istio-system
+spec:
+  podSelector:
+    matchLabels:
+      app: citadel
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/istio-system-ns: ""
+    ports:
+    - port: 8060
+    - port: 8080
+    - port: 15014
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: mixer-network-policy
+  namespace: istio-system
+spec:
+  podSelector:
+    matchLabels:
+      app: mixer
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/istio-system-ns: ""
+    ports:
+    - port: 9091
```

Comment:
Just a suggestion, not sure how it's going to look, but these port looks very cryptic. Maybe if we move them to variables and use them instead it will make this file easier to modify in the future. 

```yaml
policy_port: 15004
prometheus_port: 15090
```